XamlControlsGallery/DataModel/ControlInfoData.json

```diff
@@ -1956,10 +1956,6 @@
               "Title": "AnimatedIcon - API",
               "Uri": "https://www.docs.microsoft.com/uwp/api/microsoft.ui.xaml.controls.animatedicon"
             },
-            {
-              "Title": "AnimatedIcon - Guidance ",
-              "Uri": "https://docs.microsoft.com/windows/uwp/design/style/animated-icon"
-            },
```

Comment:
Since we have a valid link for this (https://docs.microsoft.com/windows/apps/design/controls/animated-icon), it's better to replace the link than removing that entry.

Reply:
Thanks, I didn't notice that link.
Updated.